### PR TITLE
[15.0][MIG] analytic_partner: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT44155

@pedrobaeza please review!

This PR superseeds #515

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The text for co-authorship is not correct. It should be `Co-authored-by` (it can serve any casing), not `Co-Authored By`.

https://docs.github.com/es/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Comment by stefan-tecnativa** [comment]
> The text for co-authorship is not correct. It should be `Co-authored-by` (it can serve any casing), not `Co-Authored By`.
> 
> https://docs.github.com/es/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Reviewed. Thanks for the reference.

**Comment by stefan-tecnativa** [comment]
Is this ready to merge? @pedrobaeza

File: analytic_partner/README.rst [hunk 1]
```diff
@@ -2,28 +2,31 @@
...
     :target: https://odoo-community.org/page/development-status
     :alt: Beta
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--analytic-lightgray.png?logo=github
-    :target: https://github.com/OCA/account-analytic/tree/14.0/analytic_partner
-    :alt: OCA/account-analytic
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--analityc-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-analityc/tree/16.0/analytic_partner
```
Comment (pedrobaeza) [inline analytic_partner/README.rst]: This is putting 16 instead of 15.

File: analytic_partner/__manifest__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 {
     "name": "Link analytic items and partner",
     "summary": "Search and group analytic entries by partner",
-    "version": "14.0.1.0.0",
+    "version": "15.0.0.1.0",
```
Comment (pedrobaeza) [inline analytic_partner/__manifest__.py]: Incorrect version:
```
    "version": "15.0.1.0.0",
```

File: analytic_partner/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+To configure this module, you need to:
+
+#.  Go to **Setting > Users > Groups** and check that
+    your user has the "Show Full Accounting Features" and
+    "Analytic Accounting" access rights.
```
Comment (pedrobaeza) [inline analytic_partner/readme/CONFIGURE.rst]:
```suggestion
#. Go to **Setting > Users > Groups**.
#. Search "Show Full Accounting Features".
#. Make sure your user is included in such group.
#. Do the same for "Analytic Accounting" group.
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: analytic_partner/readme/CONFIGURE.rst [hunk 4]
```diff
@@ -1,5 +1,6 @@
 To configure this module, you need to:
 
-#. Go to **Setting > Users > Users** and check you have as at least "Adviser"
-   in your user profile and have checked the "Show Full Accounting Features" and
-   "Analytic Accounting" access rights.
+#.  Go to **Setting > Users > Groups**.
+    Search "Show Full Accounting Features".
+    Make sure your user is included in such group.
+    Do the same for "Analytic Accounting" group.
```
Comment (chienandalu) [inline analytic_partner/readme/CONFIGURE.rst]: Are these supposed to numbered bullets?

Comment (pedrobaeza) [reply analytic_partner/readme/CONFIGURE.rst]: Yes, I put them that way: https://github.com/OCA/account-analytic/pull/573#discussion_r1286618068

Why removing them, @stefan-tecnativa ?

Comment (stefan-tecnativa) [reply analytic_partner/readme/CONFIGURE.rst]: I hadn't paid attention to it, sorry.

Comment (pedrobaeza) [reply analytic_partner/readme/CONFIGURE.rst]: Still incorrect, as there are too much spaces in between.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**